### ISSUE TITLE
Open nomacs in frameless mode from CLI

### ISSUE THREAD

**Issue by mmahmoudian** [nomacs/nomacs#838]
**Is your feature request related to a problem? Please describe.**
This is a great software and I would like to integrate it more with my workflow, but the UI is very busy for a one-off mode when I just want to see the image without anything extra. Nomacs at the moment has a "frameless" mode which can be toggled in using <kbd>F10</kbd>, but unfortunately it will save it in config and all later instances of nomacs will be in frameless:

![image](https://user-images.githubusercontent.com/390889/154667771-58266bd9-4fea-40ce-b41b-3a2745bf4bda.png)

**Describe the solution you'd like**
It would be nice to have a CLI argument like `nomacs --frameless image.png` to open the image in a frameless mode. At the moment upon toggling the frameless mode, user is asked to select the monitor if they have multiple monitors. This can be avoided by selecting the "active" monitor or where the mouse is. Alternatively it can be also defined as the value of that argument (e.g `nomacs --frameless=2 image.png` or `nomacs --frameless --monitor=2 image.png`)

**Describe alternatives you've considered**
Explained in the section above.

**Additional context**
This feature will be most useful for users that use TUI file managers (e.g vifm, ranger, ...).

**Comment by mmahmoudian** [comment]
Thanks for the answer.The manpage of nomacs is practically empty (I'm thinking about making a PR) and I didn't see this info in the `--help` (perhaps slipped my eye). I have to try this to see if it works as a one-off mode or doing it once would make the nomacs frameless for the future runs.

**Comment by mmahmoudian** [comment]
I just checked and there are two problems with `--mode frameless`:
1. it will ask for the monitor which is very inconvenient (would be nice to have it open in the active monitor and also to be able to specify the monitor with a CLI argument as explain in the original post
![image](https://user-images.githubusercontent.com/390889/155159792-66fb41f7-8895-4cc1-8575-832ff6253d69.png)


2. it changes the mode of nomacs forever, meaning all future instances of nomacs will be loaded in frameless mode even though the command used is `nomacs image.png`. This is also contradictory to the `--help` sections which states that `--mode` has a default mode, which means by default it should run in "framed" mode.

**Comment by scrubbbbs** [comment]
@mmahmoudian  `--mode` is used by nomacs to switch between modes, which is why it isn't a temporary change. To prevent the state change you can use `--private` which disables saving any and all settings for that session.

**Comment by scrubbbbs** [comment]
To summarize, the only feature we are missing is a way to set the monitor index from the command line.

`--mode frameless` allows you to set the mode
`--private` makes the mode change temporary
`--mode default` puts it back to normal (should you choose not to make it temporary)

**Comment by sliedes** [comment]
This is indeed confusing. I don't think it's only about being able to specify the default on the command line.

I tried nomacs, chose once "Frameless", and... I think unless I'm mistaken the only way I could find out how to access the menu again was to do a Google search for how to exit the frameless mode, since it even is remembered when the program is closed (which seems like an odd choice to me) and there's no button to exit the frameless mode in the frameless mode?

**Comment by scrubbbbs** [comment]
Yes the only way to exit frameless is the hotkey or context menu with an image open. It is unfortunate due to minimal UI frameless mode is trying to achieve. Sure there could be a floating close button in a corner but it would weaken the aesthetic. 

Also frameless is commonly used for "tracing paper" workflow e.g. you activate frameless with some transparency, then move it over your drawing program to trace it for a sketch, compare images or whatnot. Any extra UI could get in the way of trying to do something like that.

I didn't design the modes restoring behavior but I can guess how it ended up this way. Because the UI is customizable in many ways, it would be frustrating if some aspect was not restored on restart but others were. So just try to restore everything to how it was.
